org.opent2t.sample.lamp.superpopular/com.insteon.lightbulb/js/thingTranslator.js

```diff
@@ -0,0 +1,137 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
```

Comment:
Removed.

---
In reply to: [82906262](https://github.com/openT2T/translators/pull/107#discussion_r82906262) [](ancestors = 82906262)